autopush/router/simple.py

```diff
@@ -152,9 +166,21 @@ def route_notification(self, notification, uaid_data):
             returnValue(self.delivered_response(notification))
         else:
             self.metrics.increment("router.broadcast.miss")
+            if self.udp is not None:
+                yield deferToThread(self._send_wake,
+                                    self.udp)
             returnValue(self.stored_response(notification))
 
-    #############################################################
+    def _send_wake(self, wake_info):
+        if self.waker is not None:
+            try:
+                self.waker.send_wake(wake_info)
+            except WakeException:
+                raise RouterException("Could not send wakeup",
+                                      status_code=500,
+                                      response_body="Could not send Wakeup")
```

Comment:
Why does the appserver need to know this? Client wasn't connected, we couldn't deliver UDP wake, but like any other client it will eventually get its notification so we didn't error. Plus, we already stored the notification at this point, so if the appserver gets a 500, it might retry.